app/forms/new_phone_form.rb

```diff
@@ -177,4 +179,27 @@ def ingest_submitted_params(params)
   def confirmed_phone?
     false
   end
+
+  def validate_phone_submission_limit
+    fingerprint = Pii::Fingerprinter.fingerprint(Phonelib.parse(phone).e164.to_s)
+    @submission_rate_limiter ||= RateLimiter.new(
```

Comment:
If we're not using this anywhere outside the scope of this method, I don't think we need to assign it as an instance variable.

```suggestion
    submission_rate_limiter ||= RateLimiter.new(
```

Reply:
That makes sense. 👍